packages/network/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@agoric/network",
+  "version": "0.1.0",
+  "description": "Agoric's network protocol API",
+  "type": "module",
+  "main": "./src/index.js",
+  "repository": "https://github.com/Agoric/agoric-sdk",
+  "scripts": {
+    "build": "exit 0",
+    "prepack": "tsc --build jsconfig.build.json",
+    "postpack": "git clean -f '*.d.ts*'",
+    "test": "ava",
+    "test:c8": "c8 $C8_OPTIONS ava",
+    "test:xs": "exit 0",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:types": "tsc -p jsconfig.json",
+    "lint:eslint": "eslint ."
+  },
+  "keywords": [],
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@agoric/assert": "^0.6.0",
+    "@agoric/internal": "^0.3.2",
+    "@agoric/store": "^0.9.2",
+    "@agoric/swingset-vat": "^0.32.2",
+    "@endo/base64": "^0.2.32",
+    "@endo/far": "^0.2.19",
+    "@endo/promise-kit": "^0.2.57"
+  },
+  "devDependencies": {
+    "@endo/bundle-source": "^2.5.2",
+    "ava": "^5.3.0",
+    "c8": "^7.13.0"
+  },
+  "exports": {
```

Comment:
super.

i forget, does our tooling still allow deep imports anyway?

Reply:
I've found Typescript does, but Endo and Node.js definitely don't.